interface/src/app/node/node.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * "adage.gene.network" module.
+ */
+
+angular.module('adage.node', [
+  'ui.router',
+  'placeholders',
+  'ui.bootstrap'
+])
+
+.config(function config($stateProvider) {
+  $stateProvider.state('node', {
+    url: '/node/{id:int}',
+    views: {
+      main: {
+        templateUrl: 'node/node.tpl.html',
+        controller: 'NodeCtrl as ctrl'
+      }
+    },
+    data: {pageTitle: 'Node Information'}
+  });
+})
+
+.factory('NodeInfo', ['$resource', '$stateParams',
+  function($resource) {
+    return $resource('/api/v0/node/:id');
+  }
+])
+
+.factory('HeavyGenes', ['$resource', function($resource) {
+  return $resource('/api/v0/participation/');
```

Comment:
Right, I didn't see any filter here, but I wasn't sure if the `participation` back-end did any filtering and didn't dig into the code for that -- hence the caveat that this may be a dumb question.

Yes, if the back-end also does no filtering, I'd suggest that this service use the same name as the API. If the back-end *does* do filtering, then I'd suggest that the API name be changed to reflect that.